src/SFA.DAS.AANHub.Data/Configuration/CalendarEventConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Data.Configuration;
+
+public class CalendarEventConfiguration : IEntityTypeConfiguration<CalendarEvent>
```

Comment:
You can add a '[ExcludeFromCodeCoverage]' to this

Reply:
Attribute added